src/dos/dos.cpp

```diff
@@ -1204,6 +1204,25 @@ static Bitu DOS_26Handler(void) {
     return CBRET_NONE;
 }
 
+void set_ver(char *args, bool start)
+{
+	char *word = StripWord(args);
+	if (!*args && !*word) { // Reset
+		dos.version.major = 5;
+		dos.version.minor = 0;
+	} else if (*args == 0 && *word && (strchr(word, '.') != 0)) { // Allow:
+		                                                      // ver set 5.1
+		const char *p = strchr(word, '.');
+		dos.version.major = (Bit8u)(atoi(word));
```

Comment:
Use standard types `int8_u`, and C++ static_cast.

Reply:
Done.